File: src/github.com/dearcj/crossclientutils/crossclientutils.js
;(function() {
  var pkg = {};
  pkg["github.com\dearcj\crossclientutils"] = (function() {
    U.prototype.GetSkillProb = function(baseChance, will, maxwill) {
      var z = this;
      return baseChance * 1 - will / maxwill * 0.75;
    };
    function U (o) {
      o = o || {}
    };
    function main () {
      var y = new U();
      var z = y;
      y = z;
      z.GetSkillProb.bind(z)(1, 1, 1)
    };
    return {
      U: U,
      main: main
    };
  })();
  return pkg["github.com\dearcj\crossclientutils"].main();
})()

File: src/github.com/dearcj/crossclientutils/crossclientutils.go
package main

type CrossClientUtils struct{}

func (z *CrossClientUtils) GetSkillProb(baseChance float64, will float64, maxwill float64) float64 {
	return baseChance * (1 - (will/maxwill)*0.75)
}

func main() {
	var y CrossClientUtils
	var z CrossClientUtils = y
	y = z
}
